### PR TITLE
Replaced the hard-coded URLs by variables

### PR DESCRIPTION
**Pull request by ValdoTR**
Replace all the hard-coded URLs from the docker-compose file with env variables.
Add also one doc instruction about creating the env file needed by docker-compose.

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Domains
+DOMAIN=workadventure.localhost
+UPLOADER_SUBDOMAIN=pusher
```
Comment (tabascoeye) [inline .env.template:3]: should be PUSHER_SUBDOMAIN

File: docker-compose.yaml [hunk 2]
```diff
@@ -26,13 +26,13 @@ services:
       JITSI_PRIVATE_MODE: "$JITSI_PRIVATE_MODE"
       HOST: "0.0.0.0"
       NODE_ENV: development
-      API_URL: pusher.workadventure.localhost
-      UPLOADER_URL: uploader.workadventure.localhost
-      ADMIN_URL: workadventure.localhost
+      API_URL: $UPLOADER_SUBDOMAIN.$DOMAIN
```
Comment (tabascoeye) [inline docker-compose.yaml:29]: should be $PUSHER_SUBDOMAIN.$DOMAIN

File: docker-compose.yaml [hunk 3]
```diff
@@ -67,10 +67,10 @@ services:
     volumes:
       - ./pusher:/usr/src/app
     labels:
-      - "traefik.http.routers.pusher.rule=Host(`pusher.workadventure.localhost`)"
+      - "traefik.http.routers.pusher.rule=Host(`$UPLOADER_SUBDOMAIN.$DOMAIN`)"
```
Comment (tabascoeye) [inline docker-compose.yaml:70]: should be PUSHER_SUBDOMAIN

File: docker-compose.yaml [hunk 4]
```diff
@@ -67,10 +67,10 @@ services:
...
       - "traefik.http.routers.pusher.entryPoints=web"
       - "traefik.http.services.pusher.loadbalancer.server.port=8080"
-      - "traefik.http.routers.pusher-ssl.rule=Host(`pusher.workadventure.localhost`)"
+      - "traefik.http.routers.pusher-ssl.rule=Host(`$UPLOADER_SUBDOMAIN.$DOMAIN`)"
```
Comment (tabascoeye) [inline docker-compose.yaml:73]: should be PUSHER_SUBDOMAIN

**Review by tabascoeye — CHANGES_REQUESTED** [review]
looks nice, but UPLOADER should be PUSHER in a few places

**Comment by ValdoTR** [comment]
Thank you @tabascoeye ! just a copy/past issue ;)
Will fix this.

**Comment by ValdoTR** [comment]
I'm canceling thit because after thinking it, I don't think it really helps to allow the user to configure domains and subdomains, it might even mislead him.

**Pull request closed without merging**